### PR TITLE
quick and dirty fix to issue #45

### PR DESCRIPTION
**Pull request by official-sounding**
fixes Issue #45 by adding EXP-* codes to the FO_EXP list of penalties, with a subset of penalties that are expellable.

Since this is an officials best practice/paperwork requirement, I am adding this as a stopgap while I work on converting the penalty code definition to something based on a runtime-loaded file and less copy-and-paste terrible, like an "can expel" flag on a penalty definition.

### CODE REVIEW

File: src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java
Comment (brian-brazil) [inline src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java:286]: Should we add unknown too? It seems unlikely to come up, but it's possible.

Comment (JeneralPain) [reply src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java:286]: I would add unknown for the sake of it

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java:286]: Yeah, it's worth it for consistency I think.

**Review by JeneralPain — APPROVED** [review]
Approve to Merge pending Jared.

File: src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java
Comment (JeneralPain) [reply src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java:286]: Can you add it to your pull so when I merge or Jared does we can capture it?

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java:286]: I can create a quick PR after you merge for the sake of simplicity. My PR is big enough as-is.

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java:286]: Actually, better idea. https://github.com/rollerderby/scoreboard/pull/62 contains both @official-sounding's commit and this change so you can merge or rebase that one, and close this one.

**Pull request merged**
